Tools/PowershellModule/src/Microsoft.WinGet.RestSource.psd1

```diff
@@ -6,64 +6,60 @@
-    GUID                 = 'b70c845d-ddb1-4454-bfc2-a874783c2d04'
+    GUID = 'b70c845d-ddb1-4454-bfc2-a874783c2d04'
 
     # Author of this module
-    Author               = 'Microsoft Corporation'
+    Author = 'Microsoft Corporation'
 
     # Company or vendor of this module
-    CompanyName          = 'Microsoft Corporation'
+    CompanyName = 'Microsoft Corporation'
 
     # Copyright statement for this module
-    Copyright            = '(c) Microsoft Corporation. All rights reserved.'
+    Copyright = '(c) Microsoft Corporation. All rights reserved.'
 
     # Description of the functionality provided by this module
-    Description          = 'This module provides support for working with Windows Package Manager REST based sources.'
+    Description = 'This module provides support for working with Windows Package Manager REST based sources.'
 
     # Minimum version of the PowerShell engine required by this module
-    PowerShellVersion    = '7.4'
+    PowerShellVersion = '7.4'
     
     # Required modules.
     # Due to issue https://github.com/PowerShell/PowerShell/issues/11190, using RequiredModules will greatly slow down the import module. We'll handle them manually in psm1.
     # RequiredModules = @('Az')
 
     # Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
-    FunctionsToExport    = @('Add-WinGetManifest', 'Get-WinGetManifest', 'Find-WinGetManifest', 'Remove-WinGetManifest', 'New-WinGetSource', 'Update-WinGetSource')
+    FunctionsToExport = @("Add-WinGetManifest", "Get-WinGetManifest", "Find-WinGetManifest", "Remove-WinGetManifest", "New-WinGetSource", "Update-WinGetSource")
 
     # Cmdlets to export from this module, for best performance, do not use wild cards and do not delete the entry, use an empty array if there are no cmdlets to export.
-    CmdletsToExport      = @()
+    CmdletsToExport = @()
 
     # Variables to export from this module
-    VariablesToExport    = '*'
+    VariablesToExport = '*'
 
     # Aliases to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no aliases to export.
-    AliasesToExport      = @()
+    AliasesToExport = @()
 
     # Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
-    PrivateData          = @{
+    PrivateData = @{
         PSData = @{
             # Tags applied to this module. These help with module discovery in online galleries.
-            Tags       = @(
+            Tags = @(
                 'PSEdition_Core',
                 'WindowsPackageManager',
                 'WinGet'
             )
 
-            # A URL to the license for this module.
-            LicenseUri = 'https://github.com/microsoft/winget-cli-restsource/blob/main/LICENSE'
```

Comment:
I think this can be kept.

Reply:
Never mind. I checked several other winget projects. We don't need to specify a different license. We'll use the standard license by PSGallery.